js/layout-javascript/simple-list-code.js

```diff
@@ -276,6 +292,7 @@ DynamicList.prototype.attachObservers = function() {
       $parentElement.find('.hidden-filter-controls').addClass('active');
       $parentElement.find('.list-search-cancel').addClass('active');
       $elementClicked.addClass('active');
+      $parentElement.find('[data-filter-group]').prop('hidden', null);
```

Comment:
I usually use `.prop(key, false)` to disable a property, not `.prop(key, null)`. Can you confirm if there's a difference?

Reply:
In this case, there is no difference I simply took an example from this [article](https://api.jquery.com/removeAttr/)